src/Adapter/PDFLib.php

```diff
@@ -734,9 +734,7 @@ protected function _load_font($font, $encoding = null, $options = "")
         // Check if the font is a native PDF font
         // Embed non-native fonts
         $test = strtolower($baseFont);
-        if (in_array($test, DOMPDF::$nativeFonts)) {
-            $font = basename($font);
-        } else {
+        if (!in_array($test, DOMPDF::$nativeFonts)) {
```

Comment:
While at it, `$isNativeFont` is determined directly before, I think that could be reused here.